<?xml version="1.0" encoding="utf-8"?>
<feed xmlns="http://www.w3.org/2005/Atom">
    <id>https://azu.github.io/github-advisory-database-rss/pip.rss</id>
    <title>Security Advisory for Python packages hosted at PyPI.org</title>
    <updated>2024-10-27T20:01:31.912Z</updated>
    <generator>github-advisory-database-rss</generator>
    <link rel="alternate" href="https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip"/>
    <subtitle>Security Advisory for Python packages hosted at PyPI.org on GitHub</subtitle>
    <rights>github-advisory-database-rss</rights>
    <category term="CRITICAL"/>
    <category term="HIGH"/>
    <category term="MODERATE"/>
    <category term="LOW"/>
    <entry>
        <title type="html"><![CDATA[[tablib] Loaded Databook of Tablib prone to python insertion resulting in command execution]]></title>
        <id>https://github.com/advisories/GHSA-gcr6-rf47-jrgf</id>
        <link href="https://github.com/advisories/GHSA-gcr6-rf47-jrgf"/>
        <updated>2024-10-27T15:36:24.000Z</updated>
        <content type="html"><![CDATA[<p>An exploitable vulnerability exists in the Databook loading functionality of Tablib 0.11.4. A yaml loaded Databook can execute arbitrary python commands resulting in command execution. An attacker can insert python into loaded yaml to trigger this vulnerability.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2017-2810">https://nvd.nist.gov/vuln/detail/CVE-2017-2810</a></li>
<li><a href="https://security.gentoo.org/glsa/201811-18">https://security.gentoo.org/glsa/201811-18</a></li>
<li><a href="https://talosintelligence.com/vulnerability_reports/TALOS-2017-0307">https://talosintelligence.com/vulnerability_reports/TALOS-2017-0307</a></li>
<li><a href="https://github.com/jazzband/tablib/commit/69abfc3ada5d754cb152119c0b4777043657cb6e">https://github.com/jazzband/tablib/commit/69abfc3ada5d754cb152119c0b4777043657cb6e</a></li>
<li><a href="https://github.com/advisories/GHSA-gcr6-rf47-jrgf">https://github.com/advisories/GHSA-gcr6-rf47-jrgf</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/tablib/PYSEC-2017-95.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/tablib/PYSEC-2017-95.yaml</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-gcr6-rf47-jrgf</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2018-07-13T16:01:25.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[sosreport] SoSReport Predictable Tmp File Names]]></title>
        <id>https://github.com/advisories/GHSA-3g56-2hh3-35ph</id>
        <link href="https://github.com/advisories/GHSA-3g56-2hh3-35ph"/>
        <updated>2024-10-27T15:34:14.000Z</updated>
        <content type="html"><![CDATA[<p>sosreport in SoS 3.x allows local users to obtain sensitive information from sosreport files or gain privileges via a symlink attack on an archive file in a temporary directory, as demonstrated by <code>sosreport-$hostname-$date.tar</code> in <code>/tmp/sosreport-$hostname-$date</code>.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2015-7529">https://nvd.nist.gov/vuln/detail/CVE-2015-7529</a></li>
<li><a href="https://github.com/sosreport/sos/issues/696">https://github.com/sosreport/sos/issues/696</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2016:0152">https://access.redhat.com/errata/RHSA-2016:0152</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2016:0188">https://access.redhat.com/errata/RHSA-2016:0188</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=1282542">https://bugzilla.redhat.com/show_bug.cgi?id=1282542</a></li>
<li><a href="http://rhn.redhat.com/errata/RHSA-2016-0152.html">http://rhn.redhat.com/errata/RHSA-2016-0152.html</a></li>
<li><a href="http://rhn.redhat.com/errata/RHSA-2016-0188.html">http://rhn.redhat.com/errata/RHSA-2016-0188.html</a></li>
<li><a href="http://www.ubuntu.com/usn/USN-2845-1">http://www.ubuntu.com/usn/USN-2845-1</a></li>
<li><a href="https://access.redhat.com/security/cve/CVE-2015-7529">https://access.redhat.com/security/cve/CVE-2015-7529</a></li>
<li><a href="https://web.archive.org/web/20160416033632/http://www.securityfocus.com/bid/83162">https://web.archive.org/web/20160416033632/http://www.securityfocus.com/bid/83162</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/sosreport/PYSEC-2017-73.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/sosreport/PYSEC-2017-73.yaml</a></li>
<li><a href="https://github.com/advisories/GHSA-3g56-2hh3-35ph">https://github.com/advisories/GHSA-3g56-2hh3-35ph</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-3g56-2hh3-35ph</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2022-05-13T01:54:05.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[sentry] Improper authorization on debug and artifact file downloads]]></title>
        <id>https://github.com/advisories/GHSA-m4hc-m2v6-hfw8</id>
        <link href="https://github.com/advisories/GHSA-m4hc-m2v6-hfw8"/>
        <updated>2024-10-26T22:53:44.000Z</updated>
        <content type="html"><![CDATA[<h3 id="impact">Impact</h3>
<p>An authenticated user can download a debug or artifact bundle from arbitrary organizations and projects with a known bundle ID. The user does not need to be a member of the organization or have permissions on the project.</p>
<h3 id="patches">Patches</h3>
<p>A patch was issued to ensure authorization checks are properly scoped on requests to retrieve debug or artifact bundles. Authenticated users who do not have the necessary permissions on the particular project are no longer able to download them.</p>
<p><strong>Sentry SaaS users do not need to take any action. <a href="https://github.com/getsentry/self-hosted">Self-Hosted Sentry</a> users should upgrade to version 23.5.2 or higher.</strong></p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://github.com/getsentry/sentry/pull/49680">Restrict file downloads to Project</a></li>
</ul>
<h3 id="references-1">References</h3>
<ul>
<li><a href="https://github.com/getsentry/sentry/security/advisories/GHSA-m4hc-m2v6-hfw8">https://github.com/getsentry/sentry/security/advisories/GHSA-m4hc-m2v6-hfw8</a></li>
<li><a href="https://github.com/getsentry/sentry/pull/49680">https://github.com/getsentry/sentry/pull/49680</a></li>
<li><a href="https://github.com/getsentry/sentry/commit/e932b15435bf36239431eaa3790a6bcfa47046a9">https://github.com/getsentry/sentry/commit/e932b15435bf36239431eaa3790a6bcfa47046a9</a></li>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2023-36826">https://nvd.nist.gov/vuln/detail/CVE-2023-36826</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/sentry/PYSEC-2023-130.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/sentry/PYSEC-2023-130.yaml</a></li>
<li><a href="https://github.com/advisories/GHSA-m4hc-m2v6-hfw8">https://github.com/advisories/GHSA-m4hc-m2v6-hfw8</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-m4hc-m2v6-hfw8</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2023-07-25T17:19:48.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[requests-kerberos] Improper Authentication in requests-kerberos]]></title>
        <id>https://github.com/advisories/GHSA-wh37-37xw-54hr</id>
        <link href="https://github.com/advisories/GHSA-wh37-37xw-54hr"/>
        <updated>2024-10-26T22:45:18.000Z</updated>
        <content type="html"><![CDATA[<p>python-requests-Kerberos through 0.5 does not handle mutual authentication</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2014-8650">https://nvd.nist.gov/vuln/detail/CVE-2014-8650</a></li>
<li><a href="https://github.com/requests/requests-kerberos/issues/35">https://github.com/requests/requests-kerberos/issues/35</a></li>
<li><a href="https://github.com/requests/requests-kerberos/pull/36">https://github.com/requests/requests-kerberos/pull/36</a></li>
<li><a href="https://github.com/mkomitee/requests-kerberos/commit/9c1e08cc17bb6950455a85d33d391ecd2bce6eb6">https://github.com/mkomitee/requests-kerberos/commit/9c1e08cc17bb6950455a85d33d391ecd2bce6eb6</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2014-8650">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2014-8650</a></li>
<li><a href="https://github.com/requests/requests-kerberos/blob/0.6/HISTORY.rst">https://github.com/requests/requests-kerberos/blob/0.6/HISTORY.rst</a></li>
<li><a href="https://security-tracker.debian.org/tracker/CVE-2014-8650">https://security-tracker.debian.org/tracker/CVE-2014-8650</a></li>
<li><a href="http://www.openwall.com/lists/oss-security/2014/11/07/1">http://www.openwall.com/lists/oss-security/2014/11/07/1</a></li>
<li><a href="https://github.com/advisories/GHSA-wh37-37xw-54hr">https://github.com/advisories/GHSA-wh37-37xw-54hr</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/requests-kerberos/PYSEC-2019-200.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/requests-kerberos/PYSEC-2019-200.yaml</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-wh37-37xw-54hr</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2020-03-10T18:02:31.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[salt] Minion identity not validated in saltstack]]></title>
        <id>https://github.com/advisories/GHSA-jmv9-5gx8-7xpf</id>
        <link href="https://github.com/advisories/GHSA-jmv9-5gx8-7xpf"/>
        <updated>2024-10-26T22:43:28.000Z</updated>
        <content type="html"><![CDATA[<p>Salt (aka SaltStack) before 0.15.0 through 0.17.0 allows remote authenticated minions to impersonate arbitrary minions via a crafted minion with a valid key.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2013-4439">https://nvd.nist.gov/vuln/detail/CVE-2013-4439</a></li>
<li><a href="https://github.com/saltstack/salt/pull/7356">https://github.com/saltstack/salt/pull/7356</a></li>
<li><a href="http://www.openwall.com/lists/oss-security/2013/10/18/3">http://www.openwall.com/lists/oss-security/2013/10/18/3</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/salt/PYSEC-2013-14.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/salt/PYSEC-2013-14.yaml</a></li>
<li><a href="https://github.com/advisories/GHSA-jmv9-5gx8-7xpf">https://github.com/advisories/GHSA-jmv9-5gx8-7xpf</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-jmv9-5gx8-7xpf</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2022-05-17T04:58:30.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[rsa] Timing attacks in python-rsa]]></title>
        <id>https://github.com/advisories/GHSA-xrx6-fmxq-rjj2</id>
        <link href="https://github.com/advisories/GHSA-xrx6-fmxq-rjj2"/>
        <updated>2024-10-26T22:33:22.000Z</updated>
        <content type="html"><![CDATA[<p>It was found that python-rsa is vulnerable to Bleichenbacher timing attacks. An attacker can use this flaw via the RSA decryption API to decrypt parts of the cipher text encrypted with RSA</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2020-25658">https://nvd.nist.gov/vuln/detail/CVE-2020-25658</a></li>
<li><a href="https://github.com/sybrenstuvel/python-rsa/issues/165">https://github.com/sybrenstuvel/python-rsa/issues/165</a></li>
<li><a href="https://github.com/sybrenstuvel/python-rsa/commit/dae8ce0d85478e16f2368b2341632775313d41ed">https://github.com/sybrenstuvel/python-rsa/commit/dae8ce0d85478e16f2368b2341632775313d41ed</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25658</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2020:5634">https://access.redhat.com/errata/RHSA-2020:5634</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2021:0637">https://access.redhat.com/errata/RHSA-2021:0637</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2022:1716">https://access.redhat.com/errata/RHSA-2022:1716</a></li>
<li><a href="https://access.redhat.com/security/cve/CVE-2020-25658">https://access.redhat.com/security/cve/CVE-2020-25658</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=1889972">https://bugzilla.redhat.com/show_bug.cgi?id=1889972</a></li>
<li><a href="https://github.com/advisories/GHSA-xrx6-fmxq-rjj2">https://github.com/advisories/GHSA-xrx6-fmxq-rjj2</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/rsa/PYSEC-2020-100.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/rsa/PYSEC-2020-100.yaml</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA">https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP">https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7">https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/2SAF67KDGSOHLVFTRDOHNEAFDRSSYIWA</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/APF364QJ2IYLPDNVFBOEJ24QP2WLVLJP</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QY4PJWTYSOV7ZEYZVMYIF6XRU73CY6O7</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-xrx6-fmxq-rjj2</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2021-04-30T17:35:15.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[reportlab] XML Injection in ReportLab]]></title>
        <id>https://github.com/advisories/GHSA-qpg2-vx7j-3869</id>
        <link href="https://github.com/advisories/GHSA-qpg2-vx7j-3869"/>
        <updated>2024-10-26T22:32:16.000Z</updated>
        <content type="html"><![CDATA[<p>ReportLab through 3.5.26 allows remote code execution because of toColor(eval(arg)) in colors.py, as demonstrated by a crafted XML document with '&lt;span color="' followed by arbitrary Python code.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2019-17626">https://nvd.nist.gov/vuln/detail/CVE-2019-17626</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2020:0195">https://access.redhat.com/errata/RHSA-2020:0195</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2020:0197">https://access.redhat.com/errata/RHSA-2020:0197</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2020:0201">https://access.redhat.com/errata/RHSA-2020:0201</a></li>
<li><a href="https://access.redhat.com/errata/RHSA-2020:0230">https://access.redhat.com/errata/RHSA-2020:0230</a></li>
<li><a href="https://hg.reportlab.com/hg-public/reportlab/rev/51a521ad7dd3">https://hg.reportlab.com/hg-public/reportlab/rev/51a521ad7dd3</a></li>
<li><a href="https://lists.debian.org/debian-lts-announce/2020/02/msg00019.html">https://lists.debian.org/debian-lts-announce/2020/02/msg00019.html</a></li>
<li><a href="https://security.gentoo.org/glsa/202007-35">https://security.gentoo.org/glsa/202007-35</a></li>
<li><a href="https://www.debian.org/security/2020/dsa-4663">https://www.debian.org/security/2020/dsa-4663</a></li>
<li><a href="https://access.redhat.com/security/cve/cve-2019-17626">https://access.redhat.com/security/cve/cve-2019-17626</a></li>
<li><a href="https://web.archive.org/web/20191016111823/https://bitbucket.org/rptlab/reportlab/issues/199/eval-in-colorspy-leads-to-remote-code">https://web.archive.org/web/20191016111823/https://bitbucket.org/rptlab/reportlab/issues/199/eval-in-colorspy-leads-to-remote-code</a></li>
<li><a href="https://github.com/advisories/GHSA-qpg2-vx7j-3869">https://github.com/advisories/GHSA-qpg2-vx7j-3869</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/reportlab/PYSEC-2019-117.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/reportlab/PYSEC-2019-117.yaml</a></li>
<li><a href="https://usn.ubuntu.com/4273-1">https://usn.ubuntu.com/4273-1</a></li>
<li><a href="https://security.netapp.com/advisory/ntap-20240719-0006">https://security.netapp.com/advisory/ntap-20240719-0006</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZZPHP2BJSTP4IYCSJRQINP763IHO6ASL">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZZPHP2BJSTP4IYCSJRQINP763IHO6ASL</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NSCTOE3DITFICY2XKBYZ5WAF5TSQ52DM">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/NSCTOE3DITFICY2XKBYZ5WAF5TSQ52DM</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ZZPHP2BJSTP4IYCSJRQINP763IHO6ASL">https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/ZZPHP2BJSTP4IYCSJRQINP763IHO6ASL</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/NSCTOE3DITFICY2XKBYZ5WAF5TSQ52DM">https://lists.fedoraproject.org/archives/list/package-announce%40lists.fedoraproject.org/message/NSCTOE3DITFICY2XKBYZ5WAF5TSQ52DM</a></li>
<li><a href="https://bitbucket.org/rptlab/reportlab/src/default/CHANGES.md">https://bitbucket.org/rptlab/reportlab/src/default/CHANGES.md</a></li>
<li><a href="https://bitbucket.org/rptlab/reportlab/issues/199/eval-in-colorspy-leads-to-remote-code">https://bitbucket.org/rptlab/reportlab/issues/199/eval-in-colorspy-leads-to-remote-code</a></li>
<li><a href="http://lists.opensuse.org/opensuse-security-announce/2020-02/msg00002.html">http://lists.opensuse.org/opensuse-security-announce/2020-02/msg00002.html</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-qpg2-vx7j-3869</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2022-05-24T22:00:57.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[rencode] Infinite Loop in rencode]]></title>
        <id>https://github.com/advisories/GHSA-gh8j-2pgf-x458</id>
        <link href="https://github.com/advisories/GHSA-gh8j-2pgf-x458"/>
        <updated>2024-10-26T18:35:21.000Z</updated>
        <content type="html"><![CDATA[<p>The rencode package through 1.0.6 for Python allows an infinite loop in typecode decoding (such as via ;\x2f\x7f), enabling a remote attack that consumes CPU and memory.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2021-40839">https://nvd.nist.gov/vuln/detail/CVE-2021-40839</a></li>
<li><a href="https://github.com/aresch/rencode/pull/29">https://github.com/aresch/rencode/pull/29</a></li>
<li><a href="https://github.com/aresch/rencode/commit/572ff74586d9b1daab904c6f7f7009ce0143bb75">https://github.com/aresch/rencode/commit/572ff74586d9b1daab904c6f7f7009ce0143bb75</a></li>
<li><a href="https://pypi.org/project/rencode/#history">https://pypi.org/project/rencode/#history</a></li>
<li><a href="https://seclists.org/fulldisclosure/2021/Sep/16">https://seclists.org/fulldisclosure/2021/Sep/16</a></li>
<li><a href="https://github.com/advisories/GHSA-gh8j-2pgf-x458">https://github.com/advisories/GHSA-gh8j-2pgf-x458</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/rencode/PYSEC-2021-345.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/rencode/PYSEC-2021-345.yaml</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BMVQRPDVSVZNGGX57CFKCYT3DEYO4QB6">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/BMVQRPDVSVZNGGX57CFKCYT3DEYO4QB6</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MCLETLGVM5DBX6QNHQFW6TWGO5T3DENY">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MCLETLGVM5DBX6QNHQFW6TWGO5T3DENY</a></li>
<li><a href="https://security.netapp.com/advisory/ntap-20211008-0001">https://security.netapp.com/advisory/ntap-20211008-0001</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-gh8j-2pgf-x458</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2021-09-13T20:05:51.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[redis] redis-py Race Condition due to incomplete fix]]></title>
        <id>https://github.com/advisories/GHSA-8fww-64cx-x8p5</id>
        <link href="https://github.com/advisories/GHSA-8fww-64cx-x8p5"/>
        <updated>2024-10-25T21:45:10.000Z</updated>
        <content type="html"><![CDATA[<p>redis-py through 4.5.3 and 4.4.3 leaves a connection open after canceling an async Redis command at an inopportune time (in the case of a non-pipeline operation), and can send response data to the client of an unrelated request. NOTE: this issue exists because of an incomplete fix for CVE-2023-28858.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2023-28859">https://nvd.nist.gov/vuln/detail/CVE-2023-28859</a></li>
<li><a href="https://github.com/redis/redis-py/issues/2665">https://github.com/redis/redis-py/issues/2665</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2641">https://github.com/redis/redis-py/pull/2641</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2666">https://github.com/redis/redis-py/pull/2666</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2671">https://github.com/redis/redis-py/pull/2671</a></li>
<li><a href="https://github.com/redis/redis-py/releases/tag/v4.4.4">https://github.com/redis/redis-py/releases/tag/v4.4.4</a></li>
<li><a href="https://github.com/redis/redis-py/releases/tag/v4.5.4">https://github.com/redis/redis-py/releases/tag/v4.5.4</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/redis/PYSEC-2023-46.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/redis/PYSEC-2023-46.yaml</a></li>
<li><a href="https://github.com/redis/redis-py/pull/1899">https://github.com/redis/redis-py/pull/1899</a></li>
<li><a href="https://github.com/redis/redis-py/commit/66a4d6b2a493dd3a20cc299ab5fef3c14baad965">https://github.com/redis/redis-py/commit/66a4d6b2a493dd3a20cc299ab5fef3c14baad965</a></li>
<li><a href="https://github.com/redis/redis-py/commit/b3c89acd0ffe8303649ad8207bc911b1d6a033eb">https://github.com/redis/redis-py/commit/b3c89acd0ffe8303649ad8207bc911b1d6a033eb</a></li>
<li><a href="https://github.com/advisories/GHSA-8fww-64cx-x8p5">https://github.com/advisories/GHSA-8fww-64cx-x8p5</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-8fww-64cx-x8p5</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2023-03-26T21:30:23.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[redis] redis-py Race Condition due to incomplete fix]]></title>
        <id>https://github.com/advisories/GHSA-8fww-64cx-x8p5</id>
        <link href="https://github.com/advisories/GHSA-8fww-64cx-x8p5"/>
        <updated>2024-10-25T21:45:10.000Z</updated>
        <content type="html"><![CDATA[<p>redis-py through 4.5.3 and 4.4.3 leaves a connection open after canceling an async Redis command at an inopportune time (in the case of a non-pipeline operation), and can send response data to the client of an unrelated request. NOTE: this issue exists because of an incomplete fix for CVE-2023-28858.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2023-28859">https://nvd.nist.gov/vuln/detail/CVE-2023-28859</a></li>
<li><a href="https://github.com/redis/redis-py/issues/2665">https://github.com/redis/redis-py/issues/2665</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2641">https://github.com/redis/redis-py/pull/2641</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2666">https://github.com/redis/redis-py/pull/2666</a></li>
<li><a href="https://github.com/redis/redis-py/pull/2671">https://github.com/redis/redis-py/pull/2671</a></li>
<li><a href="https://github.com/redis/redis-py/releases/tag/v4.4.4">https://github.com/redis/redis-py/releases/tag/v4.4.4</a></li>
<li><a href="https://github.com/redis/redis-py/releases/tag/v4.5.4">https://github.com/redis/redis-py/releases/tag/v4.5.4</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/redis/PYSEC-2023-46.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/redis/PYSEC-2023-46.yaml</a></li>
<li><a href="https://github.com/redis/redis-py/pull/1899">https://github.com/redis/redis-py/pull/1899</a></li>
<li><a href="https://github.com/redis/redis-py/commit/66a4d6b2a493dd3a20cc299ab5fef3c14baad965">https://github.com/redis/redis-py/commit/66a4d6b2a493dd3a20cc299ab5fef3c14baad965</a></li>
<li><a href="https://github.com/redis/redis-py/commit/b3c89acd0ffe8303649ad8207bc911b1d6a033eb">https://github.com/redis/redis-py/commit/b3c89acd0ffe8303649ad8207bc911b1d6a033eb</a></li>
<li><a href="https://github.com/advisories/GHSA-8fww-64cx-x8p5">https://github.com/advisories/GHSA-8fww-64cx-x8p5</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-8fww-64cx-x8p5</uri>
        </author>
        <category label="severity" term="HIGH"/>
        <published>2023-03-26T21:30:23.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[qutebrowser] Qutebrowser XSS Vulnerability]]></title>
        <id>https://github.com/advisories/GHSA-m4fw-77v7-924m</id>
        <link href="https://github.com/advisories/GHSA-m4fw-77v7-924m"/>
        <updated>2024-10-25T21:36:49.000Z</updated>
        <content type="html"><![CDATA[<p>qutebrowser version introduced in v0.11.0 (<a href="https://github.com/qutebrowser/qutebrowser/commit/5a7869f2feaa346853d2a85413d6527c87ef0d9f">1179ee7a937fb31414d77d9970bac21095358449</a>) contains a Cross Site Scripting (XSS) vulnerability in history command, <code>qute://history</code> page that can result in Via injected JavaScript code, a website can steal the user's browsing history. This attack appear to be exploitable via the victim must open a page with a specially crafted <code>&lt;title&gt;</code> attribute, and then open the <code>qute://history</code> site via the <code>:history</code> command. This vulnerability appears to have been fixed in fixed in v1.3.3 (<a href="https://github.com/qutebrowser/qutebrowser/commit/4c9360237f186681b1e3f2a0f30c45161cf405c7">4c9360237f186681b1e3f2a0f30c45161cf405c7</a>, to be released today) and v1.4.0 (<a href="https://github.com/qutebrowser/qutebrowser/commit/5a7869f2feaa346853d2a85413d6527c87ef0d9f">5a7869f2feaa346853d2a85413d6527c87ef0d9f</a>, released later this week).</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2018-1000559">https://nvd.nist.gov/vuln/detail/CVE-2018-1000559</a></li>
<li><a href="https://github.com/qutebrowser/qutebrowser/issues/4011">https://github.com/qutebrowser/qutebrowser/issues/4011</a></li>
<li><a href="https://github.com/qutebrowser/qutebrowser/commit/4c9360237f186681b1e3f2a0f30c45161cf405c7">https://github.com/qutebrowser/qutebrowser/commit/4c9360237f186681b1e3f2a0f30c45161cf405c7</a></li>
<li><a href="https://github.com/qutebrowser/qutebrowser/commit/5a7869f2feaa346853d2a85413d6527c87ef0d9f">https://github.com/qutebrowser/qutebrowser/commit/5a7869f2feaa346853d2a85413d6527c87ef0d9f</a></li>
<li><a href="https://github.com/advisories/GHSA-m4fw-77v7-924m">https://github.com/advisories/GHSA-m4fw-77v7-924m</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/qutebrowser/PYSEC-2018-26.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/qutebrowser/PYSEC-2018-26.yaml</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-m4fw-77v7-924m</uri>
        </author>
        <category label="severity" term="MODERATE"/>
        <published>2018-09-13T15:47:57.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[pyyaml] Improper Input Validation in PyYAML]]></title>
        <id>https://github.com/advisories/GHSA-6757-jp84-gxfx</id>
        <link href="https://github.com/advisories/GHSA-6757-jp84-gxfx"/>
        <updated>2024-10-25T21:35:11.000Z</updated>
        <content type="html"><![CDATA[<p>A vulnerability was discovered in the PyYAML library in versions before 5.3.1, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. An attacker could use this flaw to execute arbitrary code on the system by abusing the python/object/new constructor.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2020-1747">https://nvd.nist.gov/vuln/detail/CVE-2020-1747</a></li>
<li><a href="https://github.com/yaml/pyyaml/pull/386">https://github.com/yaml/pyyaml/pull/386</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1747">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1747</a></li>
<li><a href="http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00017.html">http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00017.html</a></li>
<li><a href="http://lists.opensuse.org/opensuse-security-announce/2020-05/msg00017.html">http://lists.opensuse.org/opensuse-security-announce/2020-05/msg00017.html</a></li>
<li><a href="https://www.oracle.com/security-alerts/cpujul2022.html">https://www.oracle.com/security-alerts/cpujul2022.html</a></li>
<li><a href="https://github.com/yaml/pyyaml/commit/5080ba513377b6355a0502104846ee804656f1e0">https://github.com/yaml/pyyaml/commit/5080ba513377b6355a0502104846ee804656f1e0</a></li>
<li><a href="https://github.com/advisories/GHSA-6757-jp84-gxfx">https://github.com/advisories/GHSA-6757-jp84-gxfx</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/pyyaml/PYSEC-2020-96.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/pyyaml/PYSEC-2020-96.yaml</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/7PPAS6C4SZRDQLR7C22A5U3QOLXY33JX">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/7PPAS6C4SZRDQLR7C22A5U3QOLXY33JX</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/K5HEPD7LEVDPCITY5IMDYWXUMX37VFMY">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/K5HEPD7LEVDPCITY5IMDYWXUMX37VFMY</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MMQXSZXNJT6ERABJZAAICI3DQSQLCP3D">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MMQXSZXNJT6ERABJZAAICI3DQSQLCP3D</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WORRFHPQVAFKKXXWLSSW6XKUYLWM6CSH">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WORRFHPQVAFKKXXWLSSW6XKUYLWM6CSH</a></li>
<li><a href="https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZBJA3SGNJKCAYPSHOHWY3KBCWNM5NYK2">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/ZBJA3SGNJKCAYPSHOHWY3KBCWNM5NYK2</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-6757-jp84-gxfx</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2021-04-20T16:14:24.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[PyYAML] Improper Input Validation in PyYAML]]></title>
        <id>https://github.com/advisories/GHSA-8q59-q68h-6hv4</id>
        <link href="https://github.com/advisories/GHSA-8q59-q68h-6hv4"/>
        <updated>2024-10-25T21:31:46.000Z</updated>
        <content type="html"><![CDATA[<p>A vulnerability was discovered in the PyYAML library in versions before 5.4, where it is susceptible to arbitrary code execution when it processes untrusted YAML files through the full_load method or with the FullLoader loader. Applications that use the library to process untrusted input may be vulnerable to this flaw. This flaw allows an attacker to execute arbitrary code on the system by abusing the python/object/new constructor. This flaw is due to an incomplete fix for CVE-2020-1747.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2020-14343">https://nvd.nist.gov/vuln/detail/CVE-2020-14343</a></li>
<li><a href="https://github.com/yaml/pyyaml/commit/a001f2782501ad2d24986959f0239a354675f9dc">https://github.com/yaml/pyyaml/commit/a001f2782501ad2d24986959f0239a354675f9dc</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=1860466">https://bugzilla.redhat.com/show_bug.cgi?id=1860466</a></li>
<li><a href="https://github.com/yaml/pyyaml/issues/420#issuecomment-663673966">https://github.com/yaml/pyyaml/issues/420#issuecomment-663673966</a></li>
<li><a href="https://www.oracle.com/security-alerts/cpuapr2022.html">https://www.oracle.com/security-alerts/cpuapr2022.html</a></li>
<li><a href="https://www.oracle.com/security-alerts/cpujul2022.html">https://www.oracle.com/security-alerts/cpujul2022.html</a></li>
<li><a href="https://github.com/SeldonIO/seldon-core/issues/2252">https://github.com/SeldonIO/seldon-core/issues/2252</a></li>
<li><a href="https://github.com/yaml/pyyaml/issues/420">https://github.com/yaml/pyyaml/issues/420</a></li>
<li><a href="https://github.com/advisories/GHSA-8q59-q68h-6hv4">https://github.com/advisories/GHSA-8q59-q68h-6hv4</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/pyyaml/PYSEC-2021-142.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/pyyaml/PYSEC-2021-142.yaml</a></li>
<li><a href="https://pypi.org/project/PyYAML">https://pypi.org/project/PyYAML</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-8q59-q68h-6hv4</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2021-03-25T21:26:26.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[python-keystoneclient] Inadequate Encryption Strength in python-keystoneclient]]></title>
        <id>https://github.com/advisories/GHSA-c3xq-cj8f-7829</id>
        <link href="https://github.com/advisories/GHSA-c3xq-cj8f-7829"/>
        <updated>2024-10-25T21:26:24.000Z</updated>
        <content type="html"><![CDATA[<p>python-keystoneclient version 0.2.3 to 0.2.5 has middleware memcache encryption bypass.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2013-2166">https://nvd.nist.gov/vuln/detail/CVE-2013-2166</a></li>
<li><a href="https://github.com/openstack/python-keystoneclient/commit/eeefb784f24c37d5f56a421e1ccc911cace9385e">https://github.com/openstack/python-keystoneclient/commit/eeefb784f24c37d5f56a421e1ccc911cace9385e</a></li>
<li><a href="https://access.redhat.com/security/cve/cve-2013-2166">https://access.redhat.com/security/cve/cve-2013-2166</a></li>
<li><a href="https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2013-2166">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2013-2166</a></li>
<li><a href="https://bugzilla.suse.com/show_bug.cgi?id=CVE-2013-2166">https://bugzilla.suse.com/show_bug.cgi?id=CVE-2013-2166</a></li>
<li><a href="https://security-tracker.debian.org/tracker/CVE-2013-2166">https://security-tracker.debian.org/tracker/CVE-2013-2166</a></li>
<li><a href="http://lists.fedoraproject.org/pipermail/package-announce/2013-August/113944.html">http://lists.fedoraproject.org/pipermail/package-announce/2013-August/113944.html</a></li>
<li><a href="http://rhn.redhat.com/errata/RHSA-2013-0992.html">http://rhn.redhat.com/errata/RHSA-2013-0992.html</a></li>
<li><a href="http://www.openwall.com/lists/oss-security/2013/06/19/5">http://www.openwall.com/lists/oss-security/2013/06/19/5</a></li>
<li><a href="https://github.com/advisories/GHSA-c3xq-cj8f-7829">https://github.com/advisories/GHSA-c3xq-cj8f-7829</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/python-keystoneclient/PYSEC-2019-197.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/python-keystoneclient/PYSEC-2019-197.yaml</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-c3xq-cj8f-7829</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2021-10-12T16:31:43.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[python-swiftclient] Python Swift client is vulnerable to Missing SSL Certificate Check]]></title>
        <id>https://github.com/advisories/GHSA-p3xv-97g8-4wmj</id>
        <link href="https://github.com/advisories/GHSA-p3xv-97g8-4wmj"/>
        <updated>2024-10-25T21:25:11.000Z</updated>
        <content type="html"><![CDATA[<p>The OpenStack Python client library for Swift (python-swiftclient) from 1.0 before 2.0.2 does not verify X.509 certificates from SSL servers, which allows man-in-the-middle attackers to spoof servers and obtain sensitive information via a crafted certificate.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2013-6396">https://nvd.nist.gov/vuln/detail/CVE-2013-6396</a></li>
<li><a href="https://bugs.launchpad.net/python-swiftclient/+bug/1199783">https://bugs.launchpad.net/python-swiftclient/+bug/1199783</a></li>
<li><a href="http://www.openwall.com/lists/oss-security/2014/02/17/7">http://www.openwall.com/lists/oss-security/2014/02/17/7</a></li>
<li><a href="https://review.opendev.org/c/openstack/python-swiftclient/+/69187">https://review.opendev.org/c/openstack/python-swiftclient/+/69187</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/python-swiftclient/PYSEC-2014-12.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/python-swiftclient/PYSEC-2014-12.yaml</a></li>
<li><a href="https://github.com/advisories/GHSA-p3xv-97g8-4wmj">https://github.com/advisories/GHSA-p3xv-97g8-4wmj</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-p3xv-97g8-4wmj</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2022-05-17T04:52:37.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[pytorch-lightning] Code Injection in PyTorch Lightning]]></title>
        <id>https://github.com/advisories/GHSA-r5qj-cvf9-p85h</id>
        <link href="https://github.com/advisories/GHSA-r5qj-cvf9-p85h"/>
        <updated>2024-10-25T20:49:59.000Z</updated>
        <content type="html"><![CDATA[<p>PyTorch Lightning version 1.5.10 and prior is vulnerable to code injection. An attacker could execute commands on the target OS running the operating system by setting the <code>PL_TRAINER_GPUS</code> when using the <code>Trainer</code> module. A <a href="https://github.com/pytorchlightning/pytorch-lightning/commit/8b7a12c52e52a06408e9231647839ddb4665e8ae">patch</a> is included in the <code>1.6.0</code> release.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2022-0845">https://nvd.nist.gov/vuln/detail/CVE-2022-0845</a></li>
<li><a href="https://github.com/pytorchlightning/pytorch-lightning/commit/8b7a12c52e52a06408e9231647839ddb4665e8ae">https://github.com/pytorchlightning/pytorch-lightning/commit/8b7a12c52e52a06408e9231647839ddb4665e8ae</a></li>
<li><a href="https://huntr.dev/bounties/a795bf93-c91e-4c79-aae8-f7d8bda92e2a">https://huntr.dev/bounties/a795bf93-c91e-4c79-aae8-f7d8bda92e2a</a></li>
<li><a href="https://github.com/PyTorchLightning/pytorch-lightning/pull/12212">https://github.com/PyTorchLightning/pytorch-lightning/pull/12212</a></li>
<li><a href="https://github.com/advisories/GHSA-r5qj-cvf9-p85h">https://github.com/advisories/GHSA-r5qj-cvf9-p85h</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/pytorch-lightning/PYSEC-2022-181.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/pytorch-lightning/PYSEC-2022-181.yaml</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-r5qj-cvf9-p85h</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2022-03-06T00:00:16.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[python-libnmap] Argument injection in python-libnmap]]></title>
        <id>https://github.com/advisories/GHSA-qwqv-j7jr-4hp6</id>
        <link href="https://github.com/advisories/GHSA-qwqv-j7jr-4hp6"/>
        <updated>2024-10-25T20:49:08.000Z</updated>
        <content type="html"><![CDATA[<p>In the python-libnmap package through 0.7.2 for Python, remote command execution can occur (if used in a client application that does not validate arguments).</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2022-30284">https://nvd.nist.gov/vuln/detail/CVE-2022-30284</a></li>
<li><a href="https://github.com/savon-noir/python-libnmap/releases">https://github.com/savon-noir/python-libnmap/releases</a></li>
<li><a href="https://libnmap.readthedocs.io/en/latest/process.html#using-libnmap-process">https://libnmap.readthedocs.io/en/latest/process.html#using-libnmap-process</a></li>
<li><a href="https://github.com/savon-noir/python-libnmap/commit/c36fecde90017befeb4853396d0e2aac93c95b64">https://github.com/savon-noir/python-libnmap/commit/c36fecde90017befeb4853396d0e2aac93c95b64</a></li>
<li><a href="https://github.com/savon-noir/python-libnmap/releases/tag/v0.7.3">https://github.com/savon-noir/python-libnmap/releases/tag/v0.7.3</a></li>
<li><a href="https://pypi.org/project/python-libnmap">https://pypi.org/project/python-libnmap</a></li>
<li><a href="https://www.swascan.com/security-advisory-libnmap-2">https://www.swascan.com/security-advisory-libnmap-2</a></li>
<li><a href="https://github.com/pypa/advisory-database/tree/main/vulns/python-libnmap/PYSEC-2022-42999.yaml">https://github.com/pypa/advisory-database/tree/main/vulns/python-libnmap/PYSEC-2022-42999.yaml</a></li>
<li><a href="https://github.com/advisories/GHSA-qwqv-j7jr-4hp6">https://github.com/advisories/GHSA-qwqv-j7jr-4hp6</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-qwqv-j7jr-4hp6</uri>
        </author>
        <category label="severity" term="CRITICAL"/>
        <published>2022-05-06T00:00:54.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[quart] Werkzeug possible resource exhaustion when parsing file data in forms]]></title>
        <id>https://github.com/advisories/GHSA-q34m-jh98-gwm2</id>
        <link href="https://github.com/advisories/GHSA-q34m-jh98-gwm2"/>
        <updated>2024-10-25T21:35:41.000Z</updated>
        <content type="html"><![CDATA[<p>Applications using Werkzeug to parse <code>multipart/form-data</code> requests are vulnerable to resource exhaustion. A specially crafted form body can bypass the <code>Request.max_form_memory_size</code> setting.</p>
<p>The <code>Request.max_content_length</code> setting, as well as resource limits provided by deployment software and platforms, are also available to limit the resources used during a request. This vulnerability does not affect those settings. All three types of limits should be considered and set appropriately when deploying an application.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://github.com/pallets/werkzeug/security/advisories/GHSA-q34m-jh98-gwm2">https://github.com/pallets/werkzeug/security/advisories/GHSA-q34m-jh98-gwm2</a></li>
<li><a href="https://github.com/pallets/quart/commit/5e78c4169b8eb66b91ead3e62d44721b9e1644ee">https://github.com/pallets/quart/commit/5e78c4169b8eb66b91ead3e62d44721b9e1644ee</a></li>
<li><a href="https://github.com/pallets/werkzeug/commit/50cfeebcb0727e18cc52ffbeb125f4a66551179b">https://github.com/pallets/werkzeug/commit/50cfeebcb0727e18cc52ffbeb125f4a66551179b</a></li>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2024-49767">https://nvd.nist.gov/vuln/detail/CVE-2024-49767</a></li>
<li><a href="https://github.com/pallets/werkzeug/releases/tag/3.0.6">https://github.com/pallets/werkzeug/releases/tag/3.0.6</a></li>
<li><a href="https://github.com/advisories/GHSA-q34m-jh98-gwm2">https://github.com/advisories/GHSA-q34m-jh98-gwm2</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-q34m-jh98-gwm2</uri>
        </author>
        <category label="severity" term="MODERATE"/>
        <published>2024-10-25T19:44:43.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[werkzeug] Werkzeug possible resource exhaustion when parsing file data in forms]]></title>
        <id>https://github.com/advisories/GHSA-q34m-jh98-gwm2</id>
        <link href="https://github.com/advisories/GHSA-q34m-jh98-gwm2"/>
        <updated>2024-10-25T21:35:41.000Z</updated>
        <content type="html"><![CDATA[<p>Applications using Werkzeug to parse <code>multipart/form-data</code> requests are vulnerable to resource exhaustion. A specially crafted form body can bypass the <code>Request.max_form_memory_size</code> setting.</p>
<p>The <code>Request.max_content_length</code> setting, as well as resource limits provided by deployment software and platforms, are also available to limit the resources used during a request. This vulnerability does not affect those settings. All three types of limits should be considered and set appropriately when deploying an application.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://github.com/pallets/werkzeug/security/advisories/GHSA-q34m-jh98-gwm2">https://github.com/pallets/werkzeug/security/advisories/GHSA-q34m-jh98-gwm2</a></li>
<li><a href="https://github.com/pallets/quart/commit/5e78c4169b8eb66b91ead3e62d44721b9e1644ee">https://github.com/pallets/quart/commit/5e78c4169b8eb66b91ead3e62d44721b9e1644ee</a></li>
<li><a href="https://github.com/pallets/werkzeug/commit/50cfeebcb0727e18cc52ffbeb125f4a66551179b">https://github.com/pallets/werkzeug/commit/50cfeebcb0727e18cc52ffbeb125f4a66551179b</a></li>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2024-49767">https://nvd.nist.gov/vuln/detail/CVE-2024-49767</a></li>
<li><a href="https://github.com/pallets/werkzeug/releases/tag/3.0.6">https://github.com/pallets/werkzeug/releases/tag/3.0.6</a></li>
<li><a href="https://github.com/advisories/GHSA-q34m-jh98-gwm2">https://github.com/advisories/GHSA-q34m-jh98-gwm2</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-q34m-jh98-gwm2</uri>
        </author>
        <category label="severity" term="MODERATE"/>
        <published>2024-10-25T19:44:43.000Z</published>
    </entry>
    <entry>
        <title type="html"><![CDATA[[Werkzeug] Werkzeug safe_join not safe on Windows]]></title>
        <id>https://github.com/advisories/GHSA-f9vj-2wh5-fj8j</id>
        <link href="https://github.com/advisories/GHSA-f9vj-2wh5-fj8j"/>
        <updated>2024-10-26T03:47:04.000Z</updated>
        <content type="html"><![CDATA[<p>On Python &lt; 3.11 on Windows, <code>os.path.isabs()</code> does not catch UNC paths like <code>//server/share</code>. Werkzeug's <code>safe_join()</code> relies on this check, and so can produce a path that is not safe, potentially allowing unintended access to data. Applications using Python &gt;= 3.11, or not using Windows, are not vulnerable.</p>
<h3 id="references">References</h3>
<ul>
<li><a href="https://github.com/pallets/werkzeug/security/advisories/GHSA-f9vj-2wh5-fj8j">https://github.com/pallets/werkzeug/security/advisories/GHSA-f9vj-2wh5-fj8j</a></li>
<li><a href="https://github.com/pallets/werkzeug/commit/2767bcb10a7dd1c297d812cc5e6d11a474c1f092">https://github.com/pallets/werkzeug/commit/2767bcb10a7dd1c297d812cc5e6d11a474c1f092</a></li>
<li><a href="https://nvd.nist.gov/vuln/detail/CVE-2024-49766">https://nvd.nist.gov/vuln/detail/CVE-2024-49766</a></li>
<li><a href="https://github.com/pallets/werkzeug/releases/tag/3.0.6">https://github.com/pallets/werkzeug/releases/tag/3.0.6</a></li>
<li><a href="https://github.com/advisories/GHSA-f9vj-2wh5-fj8j">https://github.com/advisories/GHSA-f9vj-2wh5-fj8j</a></li>
</ul>
]]></content>
        <author>
            <name>GitHub</name>
            <email>GitHub@noreply.github.com</email>
            <uri>https://github.com/advisories/GHSA-f9vj-2wh5-fj8j</uri>
        </author>
        <category label="severity" term="MODERATE"/>
        <published>2024-10-25T19:43:41.000Z</published>
    </entry>
</feed>